master/test/streamdata/stream_trials.sql

```diff
@@ -0,0 +1,125 @@
+-- XXX: remove migration code after it makes it to an actual migration
+
+ALTER TABLE trials ADD COLUMN IF NOT EXISTS seq bigint;
+
+-- the sequence
+CREATE SEQUENCE IF NOT EXISTS stream_trial_seq START 1;
+
+-- trigger function to update sequence number on row modification
+-- this should fire BEFORE so that it can modify NEW directly.
+CREATE OR REPLACE FUNCTION stream_trial_seq_modify() RETURNS TRIGGER AS $$
+BEGIN
+    NEW.seq = nextval('stream_trial_seq');
+RETURN NEW;
+END;
+$$ LANGUAGE plpgsql;
+--
+DROP TRIGGER IF EXISTS stream_trial_trigger_seq ON trials;
+CREATE TRIGGER stream_trial_trigger_seq
+    BEFORE INSERT OR UPDATE OF
+    state, start_time, end_time, runner_state, restarts, tags
+                     ON trials
+                         FOR EACH ROW EXECUTE PROCEDURE stream_trial_seq_modify();
+
+-- helper function to create trial jsonb object for streaming
+CREATE OR REPLACE FUNCTION stream_trial_notify(
+    before jsonb, beforeproj integer, after jsonb, afterproj integer
```

Comment:
it looks like this is outdated, it should be using workspaces instead of projects